tutorials/plot_object_annotations.py

```diff
@@ -0,0 +1,141 @@
+The second element of the triplets have no direct relation between the two
+structures. For the events case, the second element corresponds to id of the
+previous active event. Whereas, the second element of the
+:class:`mne.Annotations` is a float indicating its duration in seconds.
+
+See :ref:`sphx_glr_auto_examples_io_plot_read_events.py`
+for a complete example in how to read, select and visualize **events**;
+and :ref:`sphx_glr_auto_tutorials_plot_artifacts_correction_rejection.py` to
+know how :class:`mne.Annotations` are used to mark bad segments of data.
+
+An example of events and annotations
+------------------------------------
+
+The following example shows the recorded events in `sample_audvis_raw.fif` and
+marks bad segments due to eye blinks.
+"""
+
+import os.path as op
+import numpy as np
+
+import mne
+
+# load the data
+data_path = mne.datasets.sample.data_path()
+fname = op.join(data_path, 'MEG', 'sample', 'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(fname)
+
+# plot the events
+events = mne.find_events(raw)
+
+# Specify event_id dictionary based on the experiment
+event_id = {'Auditory/Left': 1, 'Auditory/Right': 2,
+            'Visual/Left': 3, 'Visual/Right': 4,
+            'smiley': 5, 'button': 32}
+color = {1: 'green', 2: 'yellow', 3: 'red', 4: 'c', 5: 'black', 32: 'blue'}
+
+mne.viz.plot_events(events, raw.info['sfreq'], raw.first_samp, color=color,
+                    event_id=event_id)
+
+# create some annotations
+annotated_blink_raw = raw.copy()
+eog_events = mne.preprocessing.find_eog_events(raw)
+n_blinks = len(eog_events)
+# Center to cover the whole blink with full duration of 0.5s:
+onset = eog_events[:, 0] / raw.info['sfreq'] - 0.25
+duration = np.repeat(0.5, n_blinks)
+annot = mne.Annotations(onset, duration, ['bad blink'] * n_blinks,
+                        orig_time=raw.info['meas_date'])
+annotated_blink_raw.set_annotations(annot)
+
+annotated_blink_raw.plot()  # plot the annotated raw
+
+
+###############################################################################
+# Working with Annotations
+# ------------------------
+#
+# An important element of the :class:`mne.Annotations` is ``orig_time`` which
+# is the time reference for the ``onset``. It is key to understand that when
+# calling `raw.set_annotation`, the given annotations is copied and transformed
+# so that `raw.annotations.orig_time` matches the recording time of the raw
+# object. (check :class:`mne.Annotations` documentation notes to see the
+# expected behavior depending of `meas_date` and `orig_time`. Notice that
+# `meas_date` is the :class:`Info <mne.Info>` attribute of the recording time.
+# Find more in :ref:`sphx_glr_auto_tutorials_plot_info.py`)
+#
+# We'll now manipulate some simulated annotations objects.
+#
+# First let's create an annotation object without orig_time. It this case
+# one assumes that the orig_time is the time of the first sample of data.
```

Comment:
first sample of data = `raw.first_samp`? Is `raw.first_samp` same as `raw.info['meas_date']` ?